.gitignore

```diff
@@ -1,23 +1 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think this should have stayed the way that it was. There's nothing wrong with ignoring more than what you already have.